tests/gis_tests/geoapp/test_functions.py

```diff
@@ -371,6 +371,17 @@ def test_intersection(self):
             else:
                 self.assertIs(c.inter.empty, True)
 
+    @skipUnlessDBFeature("supports_empty_geometries")
```

Comment:
I can imagine a backend which supports empty geometries but doesn't have `IsEmpty` function, so I added `IsEmpty` to the `unsupported_functions`.